configs/pointnet2/metafile.yml

```diff
@@ -15,9 +15,9 @@ Collections:
       Version: v0.14.0
 
 Models:
-  - Name: pointnet2_ssg_xyz-only_16x2_cosine_200e_scannet_seg-3d-20class.py
+  - Name: pointnet2_ssg-xyz-only_2xb16-cosine-200e_scannet-seg.py
```

Comment:
xyz-only means only using the first three dimensions info during training and testing, it's more like dataset info, should we move it to `scannet-seg-xyz-only`?